uber/models/commerce.py

```diff
@@ -83,3 +85,43 @@ class StripeTransaction(MagModel):
     desc = Column(UnicodeText)
     fk_id = Column(UUID)
     fk_model = Column(UnicodeText)
+
+    def process_refund(self):
```

Comment:
I'm not 100% sure where to put this function. It might belong in the Charge class, but the Charge class is pretty complicated and it seemed a little much to put this in there. At the same time, using `self.session` on a model is an anti-pattern IIRC. So maybe it should go in the Charge class after all.

It almost seems that we should have a class that handles the "model" layer of things (which is most of what our Charge class actually does) and then another class that handles the "transaction" layer (`charge_cc` in the Charge class, and this function).

Reply:
I think this function would be better in the `Session.SessionMixin` class with the other helper methods. I don't love that pattern either, but I think we want to make sure we have a valid db session before attempting to process a refund.

As it stands, this function could potentially process a refund, but fail to make a record of it in our database if the session doesn't exist.

Reply:
OK, I'll move it there :)